src/tests/comments.test.ts

```diff
@@ -0,0 +1,96 @@
+import request from "supertest";
+import {initApp} from "../server";
+import mongoose from "mongoose";
+import {Express} from "express";
+import commentsModel from "../models/comments_model";
```

Comment:
```suggestion
import {commentModel} from "../models/comments_model";
```
would make it a named `export` instead of `default export`

so everybody will have to import it with the specified name

Reply:
5f379f80c224075f50d798149e6659a006782069